react/div-color-palette/src/main.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import ReactDOM from 'react-dom';
+import ColorDiv from './colorDiv';
+
+const App = () => {
+  const [color, setColor] = useState(null);
+  const items = []
+
+  let tinycolor = require("tinycolor2");
```

Comment:
All imports should be at the top of the file

Reply:
I know Ynon recommended TinyColor, however, I have a slightly different approach:

You don't need to use a special color library, in CSS you can define color in the HSL format:
Color, Hue, Saturation: https://css-tricks.com/hsl-hsla-is-great-for-programmatic-color-control/
As a rule of thumb, it is always better to write a few extra lines yourself than to add a package.